### ISSUE TITLE
widget.observe handlers are not functional

### ISSUE THREAD

**Issue by captainsafia** [nteract/archived-desktop-app#4645]
In the Widget Events example notebook from ipywidgets, the onClick event on the button is not functional.

**Comment by miduncan** [comment]
Hey I was able to find the cause of this (although I don't think this solves all of our event issues). You just need to call `view1.delegateEvents()` after you create a WidgetView(in this example, named `view1`). I can put a PR for this once the layout/style stuff is merged into master, because I think it should go in the create_view function I made there.

**Comment by captainsafia** [comment]
@miduncan Sure! I was able to get some events working by configuring the callbacks to use an Observable.

Shortly after getting the events to work, I ended up needing to solve the bug where stdout is not redirected to the correct output.

Feel free to submit a PR with your solution as an incremental change.

**Comment by captainsafia** [comment]
I learned a little bit more about this and turns out that these issues have to do with the fact that we don't currently have an OutputArea implementation. It is related to #4650.

**Comment by miduncan** [comment]
The `delegateEvents` change accidentally made it in #4655 since that the branch I was testing it on. Considering that the PR had a create_view rewrite, though, it wasn't that out of place. 

Let me know if there is anything else I can do to help here-- I think this is our biggest blocker for acceptance in example notebooks

**Comment by captainsafia** [comment]
@miduncan I think we can go ahead and close this. The work in #4687 allows the callback in the `observe` to be actually triggered and I haven't been able to reproduce this issue since.

We can close this and if it comes up again we can re-open the issue.
